<?xml version="1.0"?>

<launch>
    <!-- joy node -->
    <node respawn="true" pkg="joy" type="joy_node" name="joy_node" >
        <param name="dev" type="string" value="/dev/input/js0" />
        <param name="deadzone" value="0.05" />
    </node>

    <!-- Axes -->
    <param name="axis_linear" value="4" type="int"/>
    <param name="axis_angular" value="3" type="int"/>
    <param name="scale_linear" value="0.3" type="double"/>
    <param name="scale_angular" value="0.3" type="double"/>


    <!-- Joystick teleop -->
    <node pkg="lns_teleop" type="teleop_joystick" name="teleop_joystick"/>
</launch>
